c++/src/RLEv2.hh

```diff
@@ -25,6 +25,7 @@
 
 #include <vector>
 
+#define MAX_LITERAL_SIZE 512
```

Comment:
This is moved here from `c++/src/RleEncoderV2.cc`?

Reply:
Yeah, so the encoder and the decoder use the same limit.